implementations/rust/ockam/ockam/Cargo.toml

```diff
@@ -18,13 +18,22 @@ version = "0.24.0-dev"
 default = ["std", "ockam_transport_tcp", "software_vault", "noise_xx"]
 software_vault = ["ockam_vault", "ockam_vault_sync_core", "ockam_vault_sync_core/software_vault", "ockam_channel/software_vault"]
 noise_xx = ["ockam_key_exchange_xx", "ockam_channel/noise_xx"]
-std = ["ockam_node", "serde/std"]
-alloc = ["ockam_core/alloc", "serde/alloc"]
-no_std = ["ockam_core/no_std", "serde"]
+
+# Option (enabled by default): "std" enables functionality expected to
```

Comment:
Should we add some check to produce meaningful error if chosen feature set is incompatible? E.g. std&no-std enabled simultaneously makes no sense, right?

Reply:
Compile-time feature checks live in `ockam_core`:

https://github.com/ockam-network/ockam/blob/e7b7d4b2f74b487ba9a8e05a9a1a526a8fcf6079/implementations/rust/ockam/ockam_core/src/lib.rs#L19